src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
+  box-sizing: border-box;
   margin: 0;
+
+  --color: #00ACDC;
 }
+
+
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+
+}
+
+.nav {
+  position: sticky;
+  top: 0;
+ }
+
+.nav__list {
+  list-style-type: none;
```

Comment:
`list-style: none` is better solution here because it removes all list styles which can be different in different browsers and `list-style-type` removes only left dot